src/components/WalletPicker/cmp.tsx

```
@@ -22,7 +22,7 @@ export const WalletPicker = ({
       const account = await wallet.callback()
       setAccount(account)
     } catch (e: any) {
-      console.log('An error has occured', e)
+      console.log('An error was occured', e)
     }
   }
```

Wouldn't it be easier to not catch the error in the component and just let it throw? This will allow users to handle rejection by themselves